recipe/meta.yaml

```
@@ -1,56 +1,68 @@
 {% set name = "pyspnego" %}
-{% set version = "0.9.2" %}
+{% set version = "0.11.2" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 0e16cb331f1bd9c85f9b80e2be04bb5da2ecaff678cc0342f68b45e4667d7966
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 994388d308fb06e4498365ce78d222bf4f3570b6df4ec95738431f61510c971b
 
 build:
-  number: 1
+  number: 2
```

```suggestion
  number: 0
```
Reset the build number for a new version